kickstarts/base.ks.erb

```diff
@@ -1,5 +1,8 @@
 # Kickstart file automatically generated by anaconda.
+
 # Run the text based installer to see dependency info in screenshots
+text
```

Comment:
@geertj how does this impact image size?  I vaguely recall the text mode not having the ability to show some of the error/details for some failed builds.  Maybe @abellotti remembers.  We changed it to GUI installer for a reason.  If this change has minimal impact on the image size, I wouldn't want to do it.